linear_search/C/linear search.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+void main()
+{
+    int a[10], i, item,n,flag=0;
+    printf("Enter number of elements : ");
+    scanf("%d",&n);
+    printf("\nEnter elements of an array:\n");
+    for (i=0; i<n; i++)
+    {printf("Element : ");
+        scanf("%d", &a[i]);}
+    printf("\nEnter item to search: ");
+    scanf("%d", &item);
+    for (i=0; i<n; i++)
+       {

```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*